knip.ts

```diff
@@ -3,7 +3,7 @@ import type { KnipConfig } from "knip";
 const config: KnipConfig = {
   entry: ["build/index.ts"],
   project: ["src/**/*.ts"],
-  ignore: ["src/types/config.ts"],
+  ignore: ["src/utils/config.ts", "src/utils/helpers.ts", "src/types/github.ts", "src/types/webhook-events.ts"],
```

Comment:
What is the reason to ignore these files specifically?

Reply:
@gentlementlegen that's a good question and I wanted to discuss this. Knip reports them as unused files and I think they are indeed unused in this repository. I am not sure if they are / should be copied in another repository and I added them to ignore list before this is decided. If you think they can be safely deleted, we can delete the unused files.

Reply:
@gentlementlegen give me a few minutes, I will delete those files and run tests on a clean repo.

Reply:
Maybe then the way to go is to get this merged, but without ignoring them. This way, we can see from PR https://github.com/ubiquibot/permit-generation/pull/7 if they are indeed unused, in which case I'll remove them safely. Don't see much need to move them to another repo. Worst case we can find get them back with versioning

Reply:
Aha! They are used in tests, will try to add tests to Knip entry points.

Reply:
@gentlementlegen I added a Jest plugin configuration for Knip, as recommended , see commit https://github.com/ubiquibot/permit-generation/pull/9/commits/6bbe548a9cd0f0b53c2820b46ebbdfcda4939c3a and https://knip.dev/reference/plugins/jest

I also removed removed unused files, removed ignore list and all tests are still passing:

```
$ yarn knip
yarn run v1.22.19
$ knip
✂️  Excellent, Knip found no issues.
Done in 3.39s.
```

and 

```
$ yarn test
yarn run v1.22.19

$ jest
 PASS  tests/register-wallet.test.ts
 PASS  tests/generate-erc721-permit.test.ts

...

Test Suites: 4 passed, 4 total
Tests:       14 passed, 14 total
Snapshots:   0 total
Time:        5.057 s
Ran all test suites.
Done in 6.81s.
```

Reply:
If you double check that we can indeed remove the files from this repo, please resolve the comment and we can merge .